Monika After Story/game/script-topics.rpy

```diff
@@ -4560,6 +4560,25 @@ label monika_ageinf:
     m "But I don't believe that my life can be summed up in a number."
     m 5a "So I'll always love you for all eternity, [player]. Just remember that."
     return
+    
+init 5 python:
+    monika_random_topics.append('monika_fanpack')
+
+label monika_fanpack:
+    python:
+        fanpack = True
+        try:
+            renpy.file("../DDLC Fan Pack/Doki Doki Literature Club OST/Disc 2 Song Info.pdf")
+        except IOError:
+            fanpack = False
+    if fanpack:    
+        m 1b "I see that you bought the Fan Pack, [player]."
+        m 2a "You didn't have to do that, you know. Going out of your way to spend money on a free game..."
+        m 3b "Although I don't particularly like my creators, I do like that you are selfless enough to donate your own money to help others follow their dreams."
+        m 5a "That's yet another reason why I love you, [player]~"
+        return
+    else:
+        jump pick_random_topic
```

Comment:
Instead of jumping to a random topic, add some dialogue here about not having the fan pack. Even though it won't cause errors, we prefer if events are self contained (i.e.: they have a single entry and exit point and the exit is a return).